src/results.rs

```diff
@@ -62,18 +62,18 @@ pub enum EnvironmentBuildStage {
 }
 
 #[derive(Serialize)]
-pub struct SuiteExecutionReport {
+pub struct PlanExecutionReport {
     pub plan_id: String,
     pub timestamp: i64,
     pub attempts: Vec<AttemptReport>,
     pub rebot: Option<RebotOutcome>,
     pub config: AttemptsConfig,
-    pub metadata: SuiteMetadata,
+    pub metadata: PlanMetadata,
 }
 
-impl WritePiggybackSection for SuiteExecutionReport {
+impl WritePiggybackSection for PlanExecutionReport {
     fn name() -> &'static str {
-        "robotmk_suite_execution_report"
+        "robotmk_plan_execution_report"
```

Comment:
Another change, which is user facing.

Reply:
I think this should be addressed with the comment above.